plugins/pmsmart/src/main/java/org/itsallcode/whiterabbit/plugin/pmsmart/PMSmartExporter.java

```diff
@@ -44,7 +44,10 @@ public void export(ProjectReport report, ProgressMonitor progressMonitor)
         try (final var driver = webDriverFactory.createWebDriver(baseUrl))
         {
             final var weekViewPage = driver.getWeekViewPage();
-            new ExportHelper(progressMonitor, weekViewPage).export(daysToExport);
+            new ExportHelper(progressMonitor, weekViewPage)
+                    .withTransferComments(
+                            Boolean.parseBoolean(config.getOptionalValue("transfer.comments").orElse("true")))
```

Comment:
```suggestion
                            config.getOptionalValue("transfer.comments").map(Boolean::parseBoolean).orElse(true))
```
I didn't try this code, please test before using it ;)

Reply:
Yes, I will try, 
But at first glance: Much better, thanks!